Add Example

Kagre opened the pull request:
Added an example which shows all of the field types available in MS-Access 2013 so that people don't need to scour the internet to try and find documentation on how to do what pertains to their situation.

Kagre committed:
Add Example

Added an example which shows all of the field types available in MS-Access 2013 so that people don't need to scour the internet to try and find documentation on how to do what pertains to their situation.

opbld32 commented:
Docs Build status updates of commit _[7bb7987](https://github.com/Kagre/office-developer-client-docs/pull/197/commits/7bb7987912ee48302a721db8538554411d8474c6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md](https://github.com/Kagre/office-developer-client-docs/blob/patch-1/docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C18%5C1f41ed19-05df-4de3-6c32-a6870955a76e%5CPullRequest%5C201907182122503121-197%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Avmfu69%2Fqd4QPFT1nurSi3iuqy4USeM9W6Yz1FBa%2FL4%3D&st=2019-07-18T21%3A20%3A44Z&se=2019-08-18T21%3A25%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Kagre committed:
Squeeze a tad bit more in

This should now be just about everything you could do under the umbrella of CREATE TABLE

opbld32 commented:
Docs Build status updates of commit _[ebbd866](https://github.com/Kagre/office-developer-client-docs/pull/197/commits/ebbd8669316b01804647d8f07eab430bacfac53d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md](https://github.com/Kagre/office-developer-client-docs/blob/patch-1/docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C1f41ed19-05df-4de3-6c32-a6870955a76e%5CPullRequest%5C201907192102438761-197%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3Qky6TksU7%2BlaBetzO65M8mhbW05ibD%2FydtuhgQs8VI%3D&st=2019-07-19T21%3A00%3A33Z&se=2019-08-19T21%3A05%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Kagre committed:
Give the table a more descriptive name

Pun improvement aside: the double tilde helps to sort the table to the top of the alphabet which is nice for data bases that have hundreds of tables, kitsch has got to be about the best fit English adjective for this table, sync also speaks to the table's potential usage as I personally use similar tables to copy and paste data from Excel into Access, and it also demonstrates how to work with table names that are not quite so alphanumeric.

opbld30 commented:
Docs Build status updates of commit _[fcf352e](https://github.com/Kagre/office-developer-client-docs/pull/197/commits/fcf352e03100a095b02093b4a95f40232e68f2d3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md](https://github.com/Kagre/office-developer-client-docs/blob/patch-1/docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C19%5C1f41ed19-05df-4de3-6c32-a6870955a76e%5CPullRequest%5C201907192200432299-197%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=g8kWUvzR5tA77XTKmmerDyTo9edI%2F7ctTWw0jyUysHg%3D&st=2019-07-19T21%3A58%3A36Z&se=2019-08-19T22%3A03%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Fixed typos

opbld31 commented:
Docs Build status updates of commit _[671c7f2](https://github.com/Kagre/office-developer-client-docs/commits/671c7f2a97d357834469b6f59c7e55e2c22ff0a8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md](https://github.com/Kagre/office-developer-client-docs/blob/patch-1/docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C2%5C1f41ed19-05df-4de3-6c32-a6870955a76e%5CPullRequest%5C201909022031086379-197%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=P3PNzMiAH6Mbf9uQtIdac%2FqyE85DAZgj17kvdgd6G44%3D&st=2019-09-02T20%3A27%3A18Z&se=2019-10-03T20%3A32%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
Updated date

opbld30 commented:
Docs Build status updates of commit _[3de62f9](https://github.com/Kagre/office-developer-client-docs/commits/3de62f9166c0a2dfa114330c060c368f3e38bd75)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md](https://github.com/Kagre/office-developer-client-docs/blob/patch-1/docs/access/desktop-database-reference/create-table-statement-microsoft-access-sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C2%5C1f41ed19-05df-4de3-6c32-a6870955a76e%5CPullRequest%5C201909022036372523-197%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pOvr5vX2%2BGukeSPaJuMPTgV8rvOHV4%2FL32OQ2%2B%2BldQM%3D&st=2019-09-02T20%3A32%3A56Z&se=2019-10-03T20%3A37%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
